### PR TITLE
Main 1907 workload health endpoint

### PR DESCRIPTION
**Pull request by RoiGlinik**

### CODE REVIEW

File: holmes/core/supabase_dal.py [hunk 1]
```diff
@@ -113,9 +114,81 @@ def get_issue_data(self, issue_id: str) -> Optional[Dict]:
...
             .table(EVIDENCE_TABLE)
-            .select(f"*")
+            .select("*")
             .filter("issue_id", "eq", issue_id)
             .execute()
         )
-        issue_data["evidence"] = evidence.data
-        return issue_data
\ No newline at end of file
+        enrichment_blacklist = {"text_file", "alert_labels", "ai_analysis", "holmes"}
```
Comment (arikalon1) [inline holmes/core/supabase_dal.py]: Why are you also removing the alert labels? and text files?

File: holmes/core/supabase_dal.py [hunk 2]
```diff
@@ -113,9 +114,81 @@ def get_issue_data(self, issue_id: str) -> Optional[Dict]:
...
+
+            res = (
+                self.client
+                .table(EVIDENCE_TABLE)
+                .select("data, enrichment_type")
+                .in_("issue_id", unique_issues)
+                .execute()
+            )
+
+            enrichment_blacklist = {"text_file", "alert_labels", "ai_analysis", "holmes"}
```
Comment (arikalon1) [inline holmes/core/supabase_dal.py]: same

**Review by arikalon1 — APPROVED** [review]
Looks good, left a minor comment

File: holmes/core/supabase_dal.py (hunk 1 above)
Comment (RoiGlinik) [reply holmes/core/supabase_dal.py]: text files are zipped and require some extra work. also the logs are taken in the investigation so it seems redundant.
alert labels just not sure how useful they are

**Pull request merged**